### PR TITLE
Clean up lessons

### PR DESCRIPTION
**Pull request by katevoss**
Closes https://github.com/catalpainternational/canoe/issues/342

More to come
🤞🏻 I didn't break anything

### CODE REVIEW

File: src/riot/Lesson/LessonFeedback.riot.html [hunk 1]
```diff
@@ -68,22 +66,8 @@ <h3>{ TRANSLATIONS.sent() }</h3>
...
-            clickedThisLessonIsGood() {
-                this.sendFeedback("fine");
-                this.update({
-                    madeFeedbackDecision: true,
-                });
-            },
-
-            clickedThisLessonIsGreat() {
-                this.sendFeedback("great");
+            clickedThisLessonIs(rating) {
```
Comment (sandroroux) [inline src/riot/Lesson/LessonFeedback.riot.html:69]: If you go down this route, I'd enforce that `rating` is `"bad"`, `"fine"`, or `"great"`.

Something like:
```js
if (!["bad", "fine", "great"].includes(rating)) {
    throw Error('You must call this with a rating of "bad", "fine", or "great"');
}
```

File: src/riot/Lesson/LessonComplete.riot.html [hunk 2]
```diff
@@ -15,7 +14,7 @@ <h2>{ TRANSLATIONS.congrats() }</h2>
                 </template>
                 <template if="{isCourseComplete()}">
                     <div>
-                        <h2>{ TRANSLATIONS.good() }</h2>
+                        <h3>{ TRANSLATIONS.good() }</h3>
                         <p>{ lessonCompleteMessage() }</p>
                         <p if="{ courseHasExam() }">{ takeCourseExamMessage() }</p>
                     </div>
```
Comment (sandroroux) [inline src/riot/Lesson/LessonComplete.riot.html:20]: To me, this screen looks wonky. Originally, the confetti was in the top-left corner, and the text and buttons were centered(?).
image

The confetti looks so alone.

**Review by sandroroux — CHANGES_REQUESTED** [review]
Looks good! I'd check the feedback-submission code's parameters and find a home for LessonComplete's lonely confetti.

File: src/riot/Lesson/LessonComplete.riot.html (hunk 2 above)
Comment (katevoss) [reply src/riot/Lesson/LessonComplete.riot.html:20]: The poor confetti 🎊 I think what happened is that the report card svg disappeared, and in the course of cleaning up `_lesson.scss` I decided to just put the confetti on this page. I'll make the confetti be less sad and alone.

**Comment by katevoss** [comment]
@sandroroux I got a little carried away and made more changes, but I addressed your feedback too.

**Review by sandroroux — APPROVED** [review]

**Pull request merged**